slackbot-background/background.go

```diff
@@ -86,6 +86,17 @@ func InitConfig() {
 		log.Fatalf("Could not create persons api client: %s", err)
 	}
 
+	go func() {
```

Comment:
Might be nicer to use a ticker, and maybe handle os signals

```
go func() {
	sigChan := make(chan os.Signal, 1)
	signal.Notify(sigChan, syscall.SIGINT, syscall.SIGTERM)

	tk := time.NewTicker(time.Hour)
	defer tk.Stop()

	for {
		select {
		case <-tk.C:
			log.Info("Refreshing access token")
			if err = globals.personsClient.RefreshAccessToken(); err != nil {
				log.Errorf("Error refreshing persons access token: %s", err)
			}
		case <-sigChan:
			log.Println("shutdown received, stopping token refresh routine")
			return
		}
	}
}()
```

Reply:
Usually I would agree, but in this case I'm not worried about it since this will only be run within GCP cloud functions.